packages/deploy-tooling/test/unit/base/deploy.test.ts

```diff
@@ -233,6 +233,9 @@ describe('base/deploy', () => {
             });
             expect(config.createTransport).toBe(false);
             expect(mockedAdtService.createTransportRequest).toBeCalledTimes(1);
+            expect(mockedAdtService.createTransportRequest).toBeCalledWith(
+                expect.objectContaining({ description: 'Created by SAP Fiori tools for ABAP repository ~name' })
```

Comment:
We need to re-consider this sentence, as its misleading.

Reply:
Thank you, it has been updated.